lib/shoryuken/environment_loader.rb

```diff
@@ -15,9 +15,12 @@ def initialize(options)
     end
 
     def load
-      initialize_options
-      initialize_logger
-      load_rails if options[:rails]
+      if options[:rails]
+        load_rails
+      else
+        initialize_options
+        initialize_logger
```

Comment:
Sorry for slow response.
What I want to say is same as [this comment](https://github.com/phstc/shoryuken/pull/197#discussion_r63645034).